packages/localizations/src/nl-NL.ts

```diff
@@ -0,0 +1,554 @@
+import type { LocalizationResource } from '@clerk/types';
+
+const commonTexts = {
+  signIn: {
+    phoneCode: {
+      title: 'Check je telefoon',
+      subtitle: 'om verder te gaan naar {{applicationName}}',
+      formTitle: 'Verificatie code',
+      formSubtitle: 'Geef de verificatie code in die verzonden is naar je telefoonnummer',
+      resendButton: 'Verstuur code opnieuw',
+    },
+  },
+} as const;
+
+export const nlNL: LocalizationResource = {
+  socialButtonsBlockButton: 'Ga verder met {{provider|titleize}}',
+  dividerText: 'or',
+  formFieldLabel__emailAddress: 'E-mailadres',
+  formFieldLabel__emailAddresses: 'E-mailadressen',
+  formFieldLabel__phoneNumber: 'Telefoonnummer',
+  formFieldLabel__username: 'Gebruikersnaam',
+  formFieldLabel__emailAddress_phoneNumber: 'E-mailadres of telefoonnummer',
+  formFieldLabel__emailAddress_username: 'E-mailadres of gebruikersnaam',
+  formFieldLabel__phoneNumber_username: 'telefoonnummer of gebruikersnaam',
+  formFieldLabel__emailAddress_phoneNumber_username: 'E-mailadres, telefoonnummer of gebruikersnaam',
+  formFieldLabel__password: 'Wachtwoord',
+  formFieldLabel__newPassword: 'Nieuw wachtwoord',
+  formFieldLabel__confirmPassword: 'Wachtwoord bevestigen',
+  formFieldLabel__firstName: 'Voornaam',
+  formFieldLabel__lastName: 'Achternaam',
+  formFieldLabel__backupCode: 'Backupcode',
+  formFieldLabel__organizationName: 'Organisatienaam',
+  formFieldLabel__role: 'Rol',
+  formFieldInputPlaceholder__emailAddress: '',
+  formFieldInputPlaceholder__emailAddresses:
+    'Typ of plak één of meerdere emailadressen, gescheiden door spaties of komma\'s.',
+  formFieldInputPlaceholder__phoneNumber: '',
+  formFieldInputPlaceholder__username: '',
+  formFieldInputPlaceholder__emailAddress_phoneNumber: '',
+  formFieldInputPlaceholder__emailAddress_username: '',
+  formFieldInputPlaceholder__phoneNumber_username: '',
+  formFieldInputPlaceholder__emailAddress_phoneNumber_username: '',
+  formFieldInputPlaceholder__password: '',
+  formFieldInputPlaceholder__firstName: '',
+  formFieldInputPlaceholder__lastName: '',
+  formFieldInputPlaceholder__backupCode: '',
+  formFieldInputPlaceholder__organizationName: '',
+  formFieldAction__forgotPassword: 'Wachtwoord vergeten',
+  formFieldHintText__optional: 'Optioneel',
+  formButtonPrimary: 'Doorgaan',
+  signInEnterPasswordTitle: 'Vul je wachtwoord in',
+  backButton: 'Terug',
+  footerActionLink__useAnotherMethod: 'Een andere methode gebruiken',
+  badge__primary: 'Hoofd',
+  badge__thisDevice: 'Dit toestel',
```

Comment:
🤓 Not sure Dutch people refer to devices as `toestel`.

Perhaps `apparaat` would be more suitable.